kube/workspace/workspace-deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: workspace
+  name: workspace
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: workspace
+  template:
+    metadata:
+      labels:
+        app: workspace
+    spec:
+      containers:
+        - env:
+            - name: ACCOUNTS_URL
+              value: http://account
+            - name: MINIO_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: huly-secret
+                  key: MINIO_ACCESS_KEY
+            - name: MINIO_ENDPOINT
+              valueFrom:
+                configMapKeyRef:
+                  name: huly-config
+                  key: MINIO_ENDPOINT
+            - name: MINIO_SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: huly-secret
+                  key: MINIO_SECRET_KEY
+            - name: MODEL_ENABLED
+              value: '*'
+            - name: MONGO_URL
+              valueFrom:
+                configMapKeyRef:
+                  name: huly-config
+                  key: MONGO_URL
+            - name: SERVER_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: huly-secret
+                  key: SERVER_SECRET
+            - name: TRANSACTOR_URL
```

Comment:
Please also add NOTIFY_INBOX_ONLY=true

Reply:
done

Reply:
Cannot see the change there. Did you forget to commit/push?

Reply:
added now